elasticsearch/jvmopts.sls

```diff
@@ -0,0 +1,18 @@
+include:
+  - elasticsearch.service
+
+{%- set major_version = salt['pillar.get']('elasticsearch:major_version', 2) %}
+
+{%- if major_version >= 5 %}
+{%- set jvm_opts = salt['pillar.get']('elasticsearch:jvm_opts') %}
+{%- if jvm_opts %}
```

Comment:
* Combine conditionals.
* Comparison should be `==` and not `>=`.
* Set `jvm_opts` outside of conditional.
* Set default value for `jvm_opts`.